Automattic/wp-calypso
# Fix: Sites created through the ecommerce flow should be private

#### Changes proposed in this Pull Request

* There is a bug that any site created by selecting `Online Store` as site type will be public - even if the user purchases a Business plan or stays on the free plan. This PR fixes the issue.
* A site should be made public only if payment transaction for eCommerce plan is successful. If payment fails, then site should remain private(which is being fixed https://github.com/Automattic/wp-calypso/pull/37356).

#### Testing instructions



* Go to /start and select `Online Store` as site type
* Complete signup and purchase a Business plan. The site should be private - without this patch, the site would have been public.

Fixes https://github.com/Automattic/wp-calypso/issues/38371
